frontend/public/locales/en/discovery.json

```diff
@@ -32,7 +32,8 @@
     "discoveryConfig.lastActiveFilter.labelHelp": "Only discovered clusters active within this time period are found.",
     "discoveryConfig.lastActiveFilter.placeholder": "7",
     "discoveryConfig.discoveryVersions.label": "Red Hat OpenShift version",
-    "discoveryConfig.discoveryVersions.labelHelp": "Only Red Hat OpenShift clusters that match the selected versions are found.",
+    "discoveryConfig.discoveryVersions.labelHelp": "All Red Hat Openshift versions are included by default unless specified in this dropdown.",
+    "discoveryConfig.discoveryVersions.placeholder": "All available Red Hat Openshift versions are included by default",
```

Comment:
All Red Hat OpenShift versions are included by default unless specified in this drop-down menu.

Changes to product representation, "drop-down" is correct and not an accessibility issue.

Reply:
Updated!